src/System.Private.CoreLib/shared/System/Type.cs

```diff
@@ -202,7 +203,7 @@ public MethodInfo GetMethod(string name, int genericParameterCount, BindingFlags
             return GetMethodImpl(name, genericParameterCount, bindingAttr, binder, callConvention, types, modifiers);
         }
 
-        protected virtual MethodInfo GetMethodImpl(string name, int genericParameterCount, BindingFlags bindingAttr, Binder binder, CallingConventions callConvention, Type[] types, ParameterModifier[] modifiers) => throw new NotSupportedException();
+        protected virtual MethodInfo GetMethodImpl(string? name, int genericParameterCount, BindingFlags bindingAttr, Binder? binder, CallingConventions callConvention, Type[]? types, ParameterModifier[]? modifiers) => throw new NotSupportedException();
```

Comment:
`name` shouldn't be nullable here. To get a MethodImpl you actually need a name. From looking at some of the overrides they do a `Debug.Assert(name != null)`

Reply:
Good catch, I missed it